src/pages/ResearcherProfile.js

```diff
@@ -11,6 +11,7 @@ import {Storage} from '../libs/storage';
 import { Notifications, getPropertyValuesFromUser, isEmailAddress} from '../libs/utils';
 
 export default function ResearcherProfile(props) {
+  const [loading, setLoading] = useState(true);
```

Comment:
minor nit: nearly all other components with this use `isLoading` instead of `loading`, so potentially match this for consistency

Reply:
Updated to match this language!